PROJECT_LIFECYCLE.md

```diff
@@ -60,9 +60,9 @@ mandatory.
 
 Simple majority of Eiffel Community Maintainers is required for a project to be accepted to Eiffel Community.
 
-Project maintainers are expected to update [PROJECTS.md](./PROJECTS.md) by sending a pull request to community
-repository and include an entry for their project. Project creation is concluded upon acceptance of the pull
-request.
+Project maintainers are expected to update [PROJECTS.md](./PROJECTS.md) and [CONTACTS.md](./CONTACTS.md) by sending a pull request to community
+repository and include an entry for their project. Project acceptance is concluded upon acceptance of the pull
+request. When accepted, Eiffel Community organization owners create the GitHub project in accordance with the [repository checklist](https://github.com/eiffel-community/eiffel-repository-template/blob/master/repo-checklist.md) and merge the corresponding pull request.
```

Comment:
This appears to be a change in the current process. Is there any reason to not let the repo proposer drive the process?

Reply:
I'm under the impression that only org owners are able to create new repos, isn't it so?

Reply:
As I interpret https://github.com/organizations/eiffel-community/settings/member_privileges any organization member can create repos.

Reply:
Great! All the better :) I'll update the instruction.